.github/workflows/actions.yml

```diff
@@ -27,29 +27,87 @@ jobs:
         key: ${{ runner.os }}-pip-${{ hashFiles('setup.py') }}
         restore-keys: |
           ${{ runner.os }}-pip-
+    - name: Setup SSH Keys and known_hosts
+      env:
+          SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+      run: |
+          ssh-agent -a $SSH_AUTH_SOCK > /dev/null
+          ssh-add - <<< "${{ secrets.KERAS_CORE }}"
     - name: Install dependencies
+      env:
+          SSH_AUTH_SOCK: /tmp/ssh_agent.sock
       run: |
-        pip install tensorflow-cpu==2.11.0
+        pip install tensorflow==2.12.0
         pip install -e ".[tests]" --progress-bar off --upgrade
-    - name: Build custom ops for tests
-      run: |
-        python build_deps/configure.py
-        bazel build keras_cv/custom_ops:all
-        cp bazel-bin/keras_cv/custom_ops/*.so keras_cv/custom_ops/
+        git clone git@github.com:keras-team/keras-core.git
+        pip install keras-core/
+        pip install "jax[cpu]"
     - name: Test with pytest
       env:
-        TEST_CUSTOM_OPS: true
+        TEST_CUSTOM_OPS: false
       run: |
         pytest keras_cv/ --ignore keras_cv/models/legacy/ --durations 0
+  multibackend:
+    name: Test the code with Keras Core
+    strategy:
+      fail-fast: false
+      matrix:
+        backend: [tensorflow, jax, torch]
```

Comment:
Gone over the head. o_0 !

@ianstenbit 
Curious to know.
What is the purpose of this PR? Is Keras going to support Jax and Torch as the backend, just like it previously did (Theano, MXNet )?

Reply:
This is an experimental dev branch that I'm trying some things out with -- I'll share an update in a few weeks